R/bootstrap-model.R

```diff
@@ -302,7 +302,7 @@ make_boot_run <- function(mod_path, boot_args){
 
 #' Store bootstrap run details before submission
 #'
-#' @param boot_runs list of boostrap model objects created by `make_boot_run()`.
+#' @param boot_models list of boostrap model objects created by `make_boot_run()`.
 #' @inheritParams make_boot_run
```

Comment:
Man this bothers me. Cant believe R CMD Check didn't catch this

Reply:
Under newer R versions it does.  I pointed you to a build above: https://github.com/metrumresearchgroup/bbr/actions/runs/9683415641/job/26718802910